docs/UserGuide.md

```diff
@@ -43,144 +43,351 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
+> * Events can have any number of comments(including 0).
+> * If description is not entered, help message for add event will appear.
 
 Examples:
 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe t/friend e/betsycrowe@gmail.com a/Newgate Prison p/1234567 t/criminal`
+* `add ev PhotoShop Workshop s/1900 f/13-02-17 c/take laptop l/CLB`
+* `add ev Industrial Talk s/1800 f/03-05-17 e/2000 l/FoS`
+
+#### 2.2.2. Adding a deadline : `add dl`
+
+Adds a deadline to the deadline list. Undoable.<br>
+Format: `add dl DESCRIPTION b/BY_DATE [e/END_TIME] [c/COMMENTS]... [l/LOCATION]`
+
+> * `dl` is used as an acronym for "deadline"
+> * Deadlines must be added with description.
+> * By date is required, while other fields are optional.
+> * By date should have format `DD-MM-YY`, e.g. 23-01-17.
+> * End time should have format `XXXX`, e.g. 2359.
+> * Deadlines can have any number of comments(including 0).
+> * If description is not entered, help message for add deadline will appear.
+
+Examples:
 
-### 2.3. Listing all persons : `list`
+* `add dl Home Assignment 1 b/21-03-17 c/submit hardcopy l/general office`
+* `add dl Project Report b/12-04-17 e/2359 c/submit softcopy c/online discussion`
+
+#### 2.2.3. Adding a task : `add ts`
+
+Adds a task to the task list. Undoable.<br>
+Format: `add ts DESCRIPTION [c/COMMENTS]... [l/LOCATION] [p/PRIORITY]`
+
+> * `ts` is used as an acronym for "task"
+> * Tasks must be added with description.
+> * Other fields are optional.
+> * Tasks can have any number of comments(including 0).
+> * Tasks can have priority 0, 1, 2 or 3.
+> * If priority is not entered, it is set to be 0 by default.
+> * If description is not entered, help message for add task will appear.
+
+Examples:
 
-Shows a list of all persons in the address book.<br>
-Format: `list`
+* `add ts Review CS2103 c/watch webcast p/1`
+* `add ts Buy fruits l/FairPrice p/2`
 
-### 2.4. Editing a person : `edit`
+### 2.3. Listing all events/deadlines/tasks : `list`
 
-Edits an existing person in the address book.<br>
-Format: `edit INDEX [NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+Shows the lists of all events/deadlines/tasks in WhatsLeft.<br>
+Format: `list [TYPE]`
 
-> * Edits the person at the specified `INDEX`.
-    The index refers to the index number shown in the last person listing.<br>
+> * Type is optional.
+> * If type is not specified, 3 separated lists of all events, deadlines and tasks will appear.
+> * Type should be `event`, `deadline` or `task`.
+
+Examples:
+
+* `list` <br>
+  Shows 3 separated lists of all events, deadlines and tasks in WhatsLeft.
+
+* `list deadline`<br>
+  Shows the list of all deadlines in WhatsLeft.
+
+### 2.4. Editing an event/deadline/task : `edit`
+
+#### 2.4.1. Editing an event : `edit ev`
+
+Edits an existing event in WhatsLeft. Undoable.<br>
+Format: `edit ev INDEX [DESCRIPTION] [s/START_TIME] [f/START_DATE] [e/END_TIME] [t/END_DATE] [c/COMMENTS]... [l/LOCATION]`
```

Comment:
As in `add`, make your command more natural by doing away with the command switches?  Just to let you know, there is a fair bit of parsing work that we expect from the project.